packages/core/src/client.ts

```diff
@@ -112,6 +113,74 @@ export class TonClient {
         }
     }
 
+    async resolveError(functionName: string, _params: any, err: TonClientError): Promise<TonClientError> {
```

Comment:
Suggest to rename `_params` -> `params`
I put underscore because of typescript error that the identifier is unused.